apps/openmw/mwworld/player.cpp

```diff
@@ -276,6 +276,9 @@ namespace MWWorld
         mAutoMove = false;
         mForwardBackward = 0;
         mTeleported = false;
+
+        // Reset mAttackingOrSpell flag to avoid animation glitches upon reload
+        MWBase::Environment::get().getWorld()->getPlayer().setAttackingOrSpell(false);
```

Comment:
Why go through World? Why not just:

<pre>
mAttackingOrSpell = false;
</pre>

Also we should reset mCurrentCrimeId, mPaidCrimeId, mLastKnownExteriorPosition

We can also reset mSaveSkills, mSaveAttributes and mMarkedPosition, just for good measure (even if there isn't a bug resulting from not clearing them).

Alternatively, we remove the clear() method and just create a new player object instead. That's less code, but inconsistent with the way clearing works elsewhere.